### PR TITLE
Add CUDOS chain to registry

### PR DESCRIPTION
**Pull request by olliegilbey**
This PR adds the JSON and image files to the Keplr chain registry for the CUDOS chain

### CODE REVIEW

**Comment by HeesungB** [comment]
@olliegilbey Can you edit the file using the link below?
https://github.com/chainapsis/keplr-chain-registry#requirement-details

You can also use the command below to check the validation locally.
```
yarn validate cosmos/cudos.json
```

**Comment by olliegilbey** [comment]
@HeesungB , thanks for the quick response.
I should have opened this PR to `main` of our fork instead first, sorry about the noise!

Have committed again that should line things up better, the `yarn validate` passes with no errors and I've resized the images to 256 square pixels.

We are just checking the gas price values and will let you know when all is good.

Thanks again for the quick response here!

File: cosmos/cudos.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		}
+	],
+	"feeCurrencies": [
+		{
+			"coinDecimals": 18,
+			"coinDenom": "CUDOS",
+			"coinGeckoId": "cudos",
+			"coinMinimalDenom": "acudos",
+			"gasPriceStep": {
+				"low": 3300000000000,
```
Comment (jgtormo) [inline cosmos/cudos.json:41]:
```suggestion
				"low": 5000000000000,
```

**Comment by HeesungB** [comment]
@olliegilbey @jgtormo Should i merge it?

**Review by jgtormo — APPROVED** [review]

**Comment by olliegilbey** [comment]
@HeesungB , yes please, all good to go.
Thanks so much 🚀

**Comment by HeesungB** [comment]
@olliegilbey @jgtormo Thank you for contribution!

**Pull request merged**